### PR TITLE
Changes home menu screen so the GamePanels are better looking

### PR DESCRIPTION
**Pull request by WaffleBuffer**
I improved the look of the main menu. However, i don't know if git will push my modifications on css style sheet. In case it didn't do it here is what to add : 
`/* Home screen game Panes */
.gamePane {
    -fx-background-color: linear-gradient(#9898fb, #000080);
    -fx-background-radius: 30;
    -fx-background-insets: 5;
    -fx-font-size: 18pt;
    -fx-padding: 10 10 10 10;
}

.gamePane:hover {
    -fx-text-fill: black;
    -fx-background-color: linear-gradient(#98fb98, #008000);
}

/* Home screen game Panes */`

and this for every them CSS (change colors accordingly).

### CODE REVIEW

**Comment by schwabdidier** [comment]
I focus on version 1.2 expected for today and I manage your pull request

**Comment by schwabdidier** [comment]
Sorry, we focus on release version 1.2 and I forgot to manage your pull request.

**Comment by schwabdidier** [comment]
@YannRobert what do you think of this pull request ? maybe is it possible to put in elements like css and to put aside his job on navigation between games ?

@WaffleBuffer next time will made a pull request per functionalities, he learn !

**Comment by WaffleBuffer** [comment]
I already pull and merge (i hope correctly) the modifications from the main repository. Now there is a working configuration screen for magic card game. As for HomeScreen, those are Panel so we can easily add some images. Next time, i'll try to add a configuration screen on the others game. Also some teaks can be done (like how to really centers game mode panes). 
Finally, i will probably try to create an abstract configuration screen class so some of the work is less redundant.

**Comment by YannRobert** [comment]
Please grant me some time to run, test and review, maybe tonight.

Just a few thought : 

I still time to extract the changes regarding CSS in an other Pull Request. It still easy to do.

```pom.xml``` should not be changed in this Pull Request.

This branch need to be rebased with ```schwabdidier/master``` and all commits could be squashed in a single commit.
Still time to do it, something like that (non interactive way of doing it)
```
git remote add schwabdidier https://github.com/schwabdidier/GazePlay.git
git fetch schwabdidier
git reset --soft schwabdidier/master
git commit -m 'better game chooser'
git push --force
```

Pull Request should not come from your own ```master``` branch.
You should branch *before* doing your change
```
git checkout -b FixIssueXYZ
git add -u
git commit -m "fix issue #XYZ"
git push github FixIssueXYZ
```

**Comment by YannRobert** [comment]
When building the sources with ```mvn```, I get modified files because the code is not already formatted.
When submitting a PR, you need to make sure to run ```mvn clean install -T 1C``` before each commit.

For now, you need to fix the formatting in the PR by adding a new commit with the formatting.

File: gazeplay-commons/pom.xml [hunk 1]
```diff
@@ -85,7 +85,8 @@
 		<dependency>
 			<groupId>com.github.coylz</groupId>
 			<artifactId>TobiiStreamEngineForJava</artifactId>
-			<version>3.0</version>
+			<version>master-3.0-gc277290-1</version>
```
Comment (YannRobert) [inline gazeplay-commons/pom.xml]: Is this change related to the PR ?

File: gazeplay-commons/pom.xml [hunk 2]
```diff
@@ -85,7 +85,8 @@
...
+		 <type>jar</type>
```
Comment (YannRobert) [inline gazeplay-commons/pom.xml]: this is not needed, as this is the default

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java [hunk 3]
```diff
@@ -188,8 +185,13 @@ private void inc(int x, int y) {
     }
 
     public void start() {
+        
+        // Start recording events when start is called
+        gameContextScene.addEventFilter(GazeEvent.ANY, recordGazeMovements);
+        gameContextScene.addEventFilter(MouseEvent.ANY, recordMouseMovements);
 
         beginTime = System.currentTimeMillis();
+        zeroTime = System.currentTimeMillis();
```
Comment (YannRobert) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java]: is this related to the PR subject ?

**Comment by YannRobert** [comment]
I have a concern in term of user experience with the intermediate screen.
To be honest I don't like it.

Also this does make the code easier to write or read.

**Comment by YannRobert** [comment]
I think the grid menu is better than the initial checkbox.

You should have proposed just that is a small Pull Request, it could have been merged already if it were not mixed with other changes.

However, there is still an issue with CSS, some built-in themes do not support the style for the new grid menu.

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java (hunk 3 above)
Comment (WaffleBuffer) [reply gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java]: Yes, because if the zero is set in constructor, then the Stats will count the time passed in the configuration screen (which, i guess is not what we want).

File: gazeplay-commons/pom.xml (hunk 2 above)
Comment (WaffleBuffer) [reply gazeplay-commons/pom.xml]: For the modifications in pom.xml, it is not directly me because i work with netbeans. I don't know why it changes this file. Maybe i could add this in a personnal .gitignore or something ?

**Comment by YannRobert** [comment]
@WaffleBuffer Please see #204
